app/controllers/application_controller.rb

```diff
@@ -55,6 +55,12 @@ def tracker(ga_category, ga_action, ga_label, ga_value=nil)
     end
   end
 
+  def rtl_locale?(locale)
+    return true if %w(ar dv he iw fa nqo ps sd ug ur yi).include?(locale)
+    return false
+  end
+  helper_method :rtl_locale?
+
   private
```

Comment:
I tried to include every locale code that uses RTL, it would be great to get some verification here... However, I doubt we'll ever support more than a few of the more common ones, but who knows!